pkg/controllermanager/controller/add.go

```diff
@@ -65,6 +68,24 @@ func AddControllersToManager(mgr manager.Manager, cfg *config.ControllerManagerC
 		return fmt.Errorf("failed adding Quota controller: %w", err)
 	}
 
+	restConfig, err := kubernetes.RESTConfigFromClientConnectionConfiguration(&cfg.GardenClientConnection, nil, kubernetes.AuthTokenFile)
+	if err != nil {
+		return err
+	}
+
+	kubernetesClient, err := kubernetesclientset.NewForConfig(restConfig)
+	if err != nil {
+		return fmt.Errorf("failed creating kubernetes client: %w", err)
+	}
+
+	if err := (&certificatesigningrequest.Reconciler{
+		CertificatesClient:     kubernetesClient,
+		CertificatesAPIVersion: "v1",
```

Comment:
Now, that only `certificates/v1` is used, can you cleanup the field and its handling from the reconciler?